services/api/package.json

```diff
@@ -21,6 +21,7 @@
     "docs:generate": "./scripts/docs/generate"
   },
   "dependencies": {
+    "@bedrockio/chrono": "^0.6.2",
```

Comment:
not loving it, i wish we could just rely on the native methods here. but fine.

Reply:
What native methods handle time zones or a billion other things we do? Are you hoping for Temporal or something? It's coming and I'm all in on it but it's not ready yet (and even when it lands I think there will be wrapper libs for it and for good reason)